scripts/patch-staking-pool-library.js

```diff
@@ -0,0 +1,48 @@
+const fs = require('fs');
```

Comment:
leave a comment at the top to explain why it's needed pls

Reply:
Comment added.